CHANGELOG.md

```diff
@@ -126,6 +126,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([`#2367`](https://github.com/polybar/polybar/issues/2367))
 - Warning message regarding T@ in bspwm module
   ([`#2371`](https://github.com/polybar/polybar/issues/2371))
+- `polybar -m` hides physical outputs covered by monitors
```

Comment:
```suggestion
- `polybar -m` shows physical outputs, even if they are covered by monitors
```

Reply:
Hum, no, previously, `polybar -m` was showing them, now it is hiding it. `polybar -M` is unchanged.

Reply:
Yes, we generally want to describe what was broken. But maybe being more explicit is clearer here:

```suggestion
- `polybar -m` used to show both physical outputs and randr monitors, even if the outputs were covered by monitors.
```